tools/azsdk-cli/Azure.Sdk.Tools.Cli/Helpers/LabelHelper.cs

```diff
@@ -0,0 +1,114 @@
+using Azure.Sdk.Tools.Cli.Configuration;
+using Azure.Sdk.Tools.Cli.Services;
+using System.Text;
+
+namespace Azure.Sdk.Tools.Cli.Helpers
+{
+    public interface ILabelHelper
+    {
+        public LabelHelper.ServiceLabelStatus CheckServiceLabel(string csvContent, string serviceName);
+        public string CreateServiceLabel(string csvContent, string serviceLabel);
+        public string NormalizeLabel(string label);
+    }
+
+    public class LabelHelper(ILogger<LabelHelper> logger) : ILabelHelper
+    {
+
+        public enum ServiceLabelStatus
+        {
+            Exists,
+            DoesNotExist,
+            NotAServiceLabel
+        }
+
+        public ServiceLabelStatus CheckServiceLabel(string csvContent, string serviceName)
+        {
+            using var reader = new StringReader(csvContent);
+            string? line;
+            while ((line = reader.ReadLine()) != null)
+            {
+                if (string.IsNullOrWhiteSpace(line))
+                {
+                    continue;
+                }
+
+                var commaIndices = line.Split(",");
+
+                if (commaIndices.Length < 2)
+                {
+                    continue; // Skip lines that don't have at least 2 commas
```

Comment:
Code is checking for at least two tokens(so one comma) and comment says at least two commas.